app/assets/javascripts/discourse/app/models/site.js

```diff
@@ -87,9 +87,19 @@ const Site = RestModel.extend({
   trackedCategoriesList(categories) {
     const trackedCategories = [];
 
+    const uncategorizedCategoryId = Site.currentProp(
+      "uncategorized_category_id"
+    );
```

Comment:
```suggestion
    const uncategorizedCategoryId = this.uncategorized_category_id;
```
Not completely sure about this one, but `uncategorized_category_id` should be available in `this` since we're in `models/site.js`. If it's available then it'd probably make sense to remove this constant and just do `this.uncategorized_category_id` later in the `if` condition.

Reply:
Good call :+1: Have committed your suggestion.